tests/parm/diag_table/diag_table_cpld_sfs.IN

```diff
@@ -0,0 +1,311 @@
+@[SYEAR]@[SMONTH]@[SDAY].@[SHOUR]Z.@[ATMRES].64bit.non-mono
```

Comment:
It seems odd that the new variable is ``omga`` and not ``omega``?

Reply:
It is named "omga" in dynamic core.